src/Generator/TypeGenerator.php

```diff
@@ -24,18 +24,22 @@
     private $type;
 
     /**
+     * @var bool
+     */
+    private $nullable;
+
+    /**
      * @var string[]
      *
      * @link http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration
      */
-    private static $internalPhpTypes = ['int', 'float', 'string', 'bool', 'array', 'callable'];
+    private static $internalPhpTypes = ['void', 'int', 'float', 'string', 'bool', 'array', 'callable'];
 
-    // @codingStandardsIgnoreStart
     /**
      * @var string a regex pattern to match valid class names or types
      */
-    private static $validIdentifierMatcher = '/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*(\\\\[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)*$/';
-    // @codingStandardsIgnoreEnd
+    private static $validIdentifierMatcher = '/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*'
+        . '(\\\\[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)*$/';
```

Comment:
Well, if some maniac does use those class names, we better be prepared :) Thanks for noticing!